### ISSUE TITLE
Specto :: "My Specto" - "IMDb: TV Watchlist" not working. IMDb Watchlist for Movies is working fine

### ISSUE THREAD

**Issue by LJHookem14** [mrknow/filmkodi#259]
Thought last week I was able to import my TV watchlist from IMDb but yesterday and today it's not working. The IMDb Watchlist which is for Movies works fine. I've tried to import from 
TOOLS
SPECTO : LIBRARY
IMDb : IMPORT TV WATCHLIST

...and it's not importing. But my Movies import fine as I tried to add a movie and it added perfectly from IMDb.

**Comment by LJHookem14** [comment]
Well my Trakt account is authorized by Specto addon but now when I goto My Specto and Trakt Collection, there is nothing there for Movies or the Trakt TV Collection and I've went an imported both in the Tools section. Something is happening with the addon not connecting with Trakt even though it says I am. After importing I should be able to goto My Specto then Trakt Collection and see all my movies in Trakt.tv

**Comment by LJHookem14** [comment]
The only thing importing right now is the IMDb Watchlist which is just the IMDb movies watchlist. 

These don't import:
Trakt Collection (Trakt moves)
Trakt TV Collection (Trakt tv shows)
IMDb TV Watchlist (IMDb tv shows)

**Comment by LJHookem14** [comment]
Still, Trakt doesn't work after linked w/ Specto. The only thing that imports is IMDb Movies.

Cmon Mr. know. I'm sure it's just a quick update. This is my goto addon now for TV Shows especially. Specto and Salts also really play a big role in my library

**Comment by xxcriticxx** [comment]
@mrknow is usually here only on the weekends (sat + sun only!)

**Comment by LJHookem14** [comment]
Weekend has come and gone. This addon needs an update in a bad bad way!

**Comment by LJHookem14** [comment]
What's worse is the addon was updated but did nothing to fix this issue and the issue was even raised before I brought it up. 

Such a great addon too. Such a shame it won't be improved to work properly. 

Nonetheless, still good but now really drops for the experienced KODI user who uses their library and Trakt/IMDb accounts.

**Comment by xxcriticxx** [comment]
am not his keeper but weekends are usually the days he shows up

**Comment by freemanpt** [comment]
Yeah would like to see this fixed too, really important feature missing, no way to match my tvshows with library integration because my IMDB list is big. Thanks btw.

**Comment by azrin1972** [comment]
Please look at #714 - found a fix, but while waiting for the update, you'll need to do it manually
